### DISCUSSION TITLE
Faker.js update removed time.recent

### DISCUSSION THREAD

**Discussion by alex-martel** [mockoon/mockoon#1005] [Help] [answered]
Not sure if this is considered a bug of a feature so went with the feature.

Our mocks make uses of this like these to generate epoch times (long format) in the future; 
` {{add (faker 'time.recent') (multiply 3 65000)}}` this way our mocks dont have to be touched and always have valid times for our application to test with.

Unfortunately with the updates of the faker.js library that helper is not there anymore. Their documentation tells to use basic javascript `new Date().getTime()` but I do not see a way to evaluate pure javascript in mockoon. Looks like we also cannot registerHelper via handlebar.

Is there a way to have an equivalent of the above? I do not see how we can unfortunately migrate to newer versions of mockoon because of this.

Thanks.

**Comment by 255kb** [comment]
I don't think they removed it. I think it was moved to the `date` namespace. 
Could you try using `{{add (faker 'date.recent') (multiply 3 65000)}}` and see if it solves your use case?

**Reply by alex-martel** [reply]
Hi,
date.recent returns a date string like the following; '2022-01-29T06:12:12.829Z' so it is not possible to simply add a number to it see https://fakerjs.dev/api/date.html#recent

time.recent on the other hand could give an epoch long like this;`faker.time.recent() // => 1653330989968` as noted here which makes it easier to simply add seconds, minutes or hours to it via simply add; see ; https://v6.fakerjs.dev/api/time.html

please also note that time.recent uses the "now" time, and date.recent uses now-1 day by default, and its only configurable per days, not minute or seconds.

**Comment by alex-martel** [comment, marked as answer]
After digging a bit more I found out that date-fns contains non UTS-35 format, and luckily enough there's one for timestamps. So ultimately I was able to get around the issue with builtin helpers like this; `{{add (now 'T') (multiply 2 65000)}}`

Thanks

**Reply by 255kb** [reply]
Good to know. Glad you found a workaround
